### PR TITLE
Add AMP framework

### PR DESCRIPTION
**Pull request by ankitsaroha**
Adds a page for AMP framework and also includes June data for AMP

### CODE REVIEW

**Comment by ankitsaroha** [comment]
@googlebot I signed it!

File: src/page-data/may-2020/amp.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export const data = {
     },
     'fcpCrux': {
       numOrigins: 5632,
-      data: [34, 20, 46],
+      data: [44, 28, 28],
```
Comment (housseindjirdeh) [inline src/page-data/may-2020/amp.js:65]: Did you run the query for May and notice this error? Thanks for spotting + fixing!!

Comment (housseindjirdeh) [reply src/page-data/may-2020/amp.js:65]: Or is it a mistake? Because it looks the same as june-2020 🤔

Comment (ankitsaroha) [reply src/page-data/may-2020/amp.js:65]: I checked the data for May, 2020 and June, 2020 and it's similar in case of FCP

**Pull request merged**